types/fxjs/tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Why did you need to disable all those lint rules?

Reply:
okay, I changed the rule to true